src/index.js

```diff
@@ -0,0 +1,140 @@
+    const DEFAULT_TEMP = 70;
+
+    cityDisplay.textContent = "Miami";
+    cityInput.addEventListener('input', () => {
+      cityDisplay.textContent = cityInput.value.trim();
+    });
+
+    landscape.style.fontFamily = "monospace";
+    landscape.style.whiteSpace = "pre";
+
+    let temperature = DEFAULT_TEMP;
+    const PROXY_URL = "http://localhost:5000";
+
+    const getRealTimeTemperature = async () => {
+      const city = cityDisplay.textContent.trim();
+  
+      if (!city) {
+        alert("Please enter a valid city name.");
+        return;
+      }
+
+      try {
+        const locationResponse = await axios.get(`${PROXY_URL}/location`, {
+          params: { q: city }
+        });
+
+        if (!locationResponse.data || locationResponse.data.length === 0) {
+          alert('Unable to find location.');
+          return;
+        }
+
+        const { lat, lon } = locationResponse.data[0];
+        const weatherResponse = await axios.get(`${PROXY_URL}/weather`, {
+          params: { lat: lat, lon: lon }
+        });
+
+        if (weatherResponse.data && weatherResponse.data.main) {
+       
+          const kelvinTemp = weatherResponse.data.main.temp;
+          temperature = Math.round((kelvinTemp - 273.15) * 9/5 + 32);
+          updateDisplay();
+        } else {
+          alert('Unable to fetch weather data.');
+        }
+      } catch (error) {
+        console.error('Error fetching data:', error.message);
+        alert('Unable to fetch temperature. Please try again.');
+      }
+    };
+
+    function updateDisplay() {
+      const tempDisplay = document.getElementById('tempValue');
+      tempDisplay.textContent = `${temperature}°F`;
+      if (temperature >= 80) {
+        tempDisplay.style.backgroundColor = "red";
+        landscape.textContent = "🌡️ BURNING HOT 🌡️\n" +
+          "  ╭──────╮   \n" +
+          " ░│ ☀️ ☀️ │░  \n" +
+          "░░╰──────╯░░\n";
+      } else if (temperature >= 70 && temperature < 80) {
+        tempDisplay.style.backgroundColor = "orange";
+        landscape.textContent = "⛵ SAILING DAY ⛵\n" +
+          "   ⛵ ⛵      \n" +
+          " ∿∿∿∿∿∿     \n" +
+          "∿∿∿∿∿∿∿∿∿\n";
+      } else if (temperature >= 60 && temperature < 70) {
+        tempDisplay.style.backgroundColor = "yellow";
+        landscape.textContent = "⛅ IDEAL DAY ⛅\n" +
+          "   ╭∩∩∩╮    \n" +
+          " ╭∩∩∩∩∩╮   \n" +
+          "∽∽∽∽∽∽∽∽∽\n";
+      } else if (temperature >= 50 && temperature < 60) {
+        tempDisplay.style.backgroundColor = "green";
+        landscape.textContent = "🍂 CRISP DAY 🍁\n" +
+          "  ≈╮__╭≈    \n" +
+          "  ⋯╭━━╮⋯   \n" +
+          " ⋯⋯○⋯⋯⋯   \n" +
+          "⋯⋯⋯⋯⋯⋯⋯\n";
+      } else if (temperature <= 49) {
+        tempDisplay.style.backgroundColor = "teal";
+        landscape.textContent = "🌨️ SNOW DAY ⛄\n" +
+          "  ╭❅╮╭❅╮   \n" +
+          " ╭❅╮╭❅╮╭❅╮  \n" +
+          "▓▓▓▓▓▓▓▓▓\n";
+      }
```

Comment:
When we have large if-statements like this where each if-block performs the same operations with slightly different values, it can be worth making a helper function that handles the updates to reduce places we're repeating code and could accumulate small typos. With the helper function, each of the ifs here would call that function passing the desired parameters rather than setting attributes themselves.